src/main/scala/com/cognite/sdk/scala/v1/fdm/instances/InstancePropertyValue.scala

```diff
@@ -110,32 +112,14 @@ object InstancePropertyValue {
         }
         numericInstantPropType.map(Right(_))
       case v if v.isBoolean => v.asBoolean.map(s => Right(InstancePropertyValue.Boolean(s)))
+      case v if v.isObject && isDirectRelation(v) =>
+        Some(decodeDirectRelation(v, c) match {
+          case Left(d) => Left(d)
+          case Right(d: DirectRelationReference) =>
+            Right(InstancePropertyValue.ViewDirectNodeRelation(Some(d)))
+        })
```

Comment:
nit: looks equivalent to
```suggestion
        Some(
            decodeDirectRelation(v, c)
                .map(d => InstancePropertyValue.ViewDirectNodeRelation(Some(d))
        )
```